src/deltacode/__init__.py

```diff
@@ -312,3 +322,48 @@ def to_dict(self):
             ('new', new_file),
             ('old', old_file),
         ])
+
+class Stat(object):
+    """
+    Contains all the stats for the file changes in the new directory
+    with respect to the old directory.
+    """
+    def __init__(self, new_files_count, old_files_count):
+        self.new_files_count = new_files_count
+        self.old_files_count = old_files_count
+        self.added = 0
+        self.removed = 0
+        self.moved = 0
+        self.modified = 0
+        self.unmodified = 0
+        self.added_percent = 0
+        self.removed_percent = 0
+        self.moved_percent = 0
+        self.modified_percent = 0
+        self.unmodified_percent = 0
+
+    def calculate_stats(self):
+        """
+        Calculates the percentage change of new directory with respect to
+        the old directory.
+        """
+        self.added_percent = percent(self.added, self.old_files_count)
+        self.removed_percent = percent(self.removed, self.old_files_count)
+        self.moved_percent = percent(self.moved, self.old_files_count)
+        self.modified_percent = percent(self.modified, self.old_files_count)
+        self.unmodified_percent = percent(self.unmodified, self.old_files_count)
+
+    def percent_to_dict(self):
```

Comment:
Probably rename this to simply `to_dict`, since this is a `Stat` object.

Reply:
@majurg I named it as `percent_to_dict` because we currently need only percent stats, in future if we want to get all the data variables of this class then we can use another method `to_dict()` for that case. Should I rename this to `to_dict()` ?

Reply:
Yes, I would rename this.